index.js

```diff
@@ -117,6 +123,27 @@ ${source.split('\n').map((line, n) => (n + 1) + '  ' + line).join('\n')}
   ])
 }
 
+const evalSpecialSol = (command, commands, options={}) => {
+  const props = specialCommands[command]
+  const struct = {}
+  let index = 0
+
+  return new Bluebird((resolve, reject) => {
+    props.forEach(prop => {
+    let statement = command + "." + prop + ";"
+    evalSol(commands.concat(statement), options)
```

Comment:
The `evalSol` command is quite slow, so I don't think it's viable to call it once for each `prop` for performance reasons.

I would suggest evaluating in a single, combined command, returning a tuple, and then unpacking the tuple into a readable struct.

This should also allow us to reuse `evalSol` without another version of it.